pkg/router/router.go

```diff
@@ -39,7 +41,7 @@ func (dr *DumbRouter) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	dr.SetHeaders(w)
 
 	if r.Method != "HEAD" && r.Method != "GET" {
-		http.Error(w, "Method Not Allowed", 405)
+		http.Error(w, http.StatusText(http.StatusMethodNotAllowed), http.StatusMethodNotAllowed)
```

Comment:
🤔 
Will the compiler optimize this to inline the string here? Adding the overhead of a method call on a bad request here would be unfortunate.

Reply:
Or is this a type conversion?... still an interesting question about inlining...

Reply:
I'm not entirely sure but I can find out. This is more of a personal taste for consistency and not having a bunch of variations with HTTP status messages throughout the app.

Reply:
ok. Please leave the current strings as they are then, and put any potentials cleanups like this into a new PR.

Reply:
🆒 reverted them in 75894d0